### PR TITLE
Feature/22213 improve unit test coverage

### PR DESCRIPTION
**Pull request by Siddartha-tech**
Improve Unit test coverage

### CODE REVIEW

**Review by barrie-cooper — COMMENTED** [review]
Thanks for these changes.  Just one minor comment that needs addressing please 😃

File: UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common.UnitTests/Helpers/FileShareServiceCacheTest.cs [hunk 1]
```diff
@@ -150,6 +151,19 @@ public async Task WhenGetNonCachedProductDataForFssIsCalled_ThenReturnProductFou
             Assert.AreEqual(1, response.Count);           
         }
 
+        [Test]
+        public async Task WhenCopyFileToBlobForCacheIsCalled_ThenReturnVoid()
+        {
+            var _cloudBlob = A.Fake<CloudBlockBlob>(o => o.WithArgumentsForConstructor(() => new CloudBlockBlob(new Uri("http://tempuri.org/blob"))));
+            A.CallTo(() => fakeAzureStorageService.GetStorageAccountConnectionString(A<string>.Ignored, A<string>.Ignored)).Returns(GetStorageAccountConnectionStringAndContainerName().Item1);
+            A.CallTo(() => fakeAzureBlobStorageClient.GetCloudBlockBlob(A<string>.Ignored, A<string>.Ignored, A<string>.Ignored)).Returns(_cloudBlob);
+            A.CallTo(() => _cloudBlob.ExistsAsync()).Returns(false);
```
[the remark this answers is not in the document]
Comment (Siddartha-tech) [reply UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common.UnitTests/Helpers/FileShareServiceCacheTest.cs]: Yes. Did changes accordingly. Thanks!

**Review by barrie-cooper — COMMENTED** [review]
Unit test needs to be improved

File: UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common.UnitTests/Helpers/FileShareServiceCacheTest.cs [hunk 2]
```diff
@@ -150,6 +151,21 @@ public async Task WhenGetNonCachedProductDataForFssIsCalled_ThenReturnProductFou
...
+        {
+            var _cloudBlob = A.Fake<CloudBlockBlob>(o => o.WithArgumentsForConstructor(() => new CloudBlockBlob(new Uri("http://tempuri.org/blob"))));
+            A.CallTo(() => fakeAzureStorageService.GetStorageAccountConnectionString(A<string>.Ignored, A<string>.Ignored)).Returns(GetStorageAccountConnectionStringAndContainerName().Item1);
+            A.CallTo(() => fakeAzureBlobStorageClient.GetCloudBlockBlob(A<string>.Ignored, A<string>.Ignored, A<string>.Ignored)).Returns(_cloudBlob);
+            A.CallTo(() => _cloudBlob.ExistsAsync()).Returns(state);
+
+            await fileShareServiceCache.CopyFileToBlob(new MemoryStream(), string.Empty, string.Empty);
+
+            Assert.IsNotNull(true);
+        }
```
[the remark this answers is not in the document]
Comment (Siddartha-tech) [reply UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common.UnitTests/Helpers/FileShareServiceCacheTest.cs]: Thanks! for suggesting improvement for unit test.

**Review by barrie-cooper — APPROVED** [review]
LGTM

**Pull request merged**